reports/hemang.md

```diff
@@ -216,11 +216,11 @@ It was a great learning experience overall, but a few challenges which proved to
 
 #### Work to be done
 
-The first task, is to get all the supporting tools like coala Online, coala-html and projects.coala.io need to incooperate user feedback on the changes user experience. 
+The first task, is to get all the supporting tools like coala Online, coala-html and projects.coala.io need to incooperate user feedback on the changes user experience.
```

Comment:
Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.